src/store/apis/users.ts

```diff
@@ -3,6 +3,9 @@ import { secretIp } from '../../../secrets/secretStuff';
 import { extractUserId } from '../../shared/utilities';
```

Comment:
i merged latest `develop` into my branch, but not sure why some of this logic is showing as a diff